### DISCUSSION TITLE
How can I use winUI elements in WPF?

### DISCUSSION THREAD

**Discussion by samsunlu55atakum** [dotnet/wpf#8147] [Q&A]
How can I use winUI elements in WPF?such as api, control, some element such as button,radiobutton...

**Comment by baochenw** [comment]
I think XAML Island can be used to host a Universal Windows Platform (UWP) XAML control. 
https://learn.microsoft.com/en-us/windows/apps/desktop/modernize/host-standard-control-with-xaml-islands
